packages/core/src/engine/verbs/fetch.ts

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+
+import { loadCSV } from 'arquero'
+import ColumnTable from 'arquero/dist/types/table/column-table'
+
+/**
+ * Executes an arquero loadCSV
+ * @param step
+ * @param store
+ * @returns
+ */
+export async function fetch(
+	url: string,
```

Comment:
I would use the same signature as the other verbs here (step, store) so that they are chainable in a pipeline.  You should have a FetchStep with FetchArgs, where the FetchArgs contains a require url field and an optional delimiter field